dependencies.yaml

```diff
@@ -84,6 +84,7 @@ devel_no_selenium_requires:
 devel_requires:
   '%devel_no_selenium_requires':
   chromedriver:
+  shfmt:
```

Comment:
I think in this case it has to be put under `test_requires`, where yamllint is listed.

Reply:
done